com/course/zenkouv/src/main/java/lecture5/Calculator.java

```diff
@@ -0,0 +1,51 @@
+package lecture5;
+
+import lecture5.util.operation.Operation;
+import lecture5.util.reader.Reader;
+
+public class Calculator {
+
+    private static final Operation OPERATION = new Operation();
+
+    public void run() {
+        double variable1 = Reader.readNumber();
+        String sign = Reader.readSign();
+        double variable2 = readSecondVariable(sign);
+        switch (sign) {
+            case "+":
+                printResult(variable1, sign, variable2, OPERATION.addition(variable1, variable2));
+                break;
+            case "-":
+                printResult(variable1, sign, variable2, OPERATION.subtraction(variable1, variable2));
+                break;
+            case "/":
+                printResult(variable1, sign, variable2, OPERATION.division(variable1, variable2));
```

Comment:
catch exception?

Reply:
i can't catch ArithmeticException because i have methods:
`private double readSecondVariable(String sign) {`
        `double variable;`
        `do {`
           ` variable = Reader.readNumber();`
          `  if (isNotZero(variable, sign)) {`
              `  return variable;`
         `   } else {`
             `   System.out.println("You try to separate by zero!");`
          `  }`
       ` } while (true);`
   ` }`